docs_app/content/tutorial/1-subscribing.md

```diff
@@ -0,0 +1,66 @@
+# 1. Subscribing to an Observable
+
+## TLDR:
+
+- Observables don't do anything until you call [subscribe](API)
+- Calling [subscribe](API) will synchronously execute code that sets up the subscription, telling the underlying [producer](GL) to start pushing values to your code (the [consumer](GL)).
+- [subscribe](API) may be called with _one function_ that is the [next](GL) handler, *OR* an [`Observer`](API) object, which has a [next](GL), [error](GL), and/or [complete](GL) handler on it.
+- If no [error](GL) handler is provided, RxJS will assume all errors that are pushed are "unhandled" errors, and rethrow them on a different call stack.
+
+## Overview
+
+Chances are, the first time you encounter an [Observable](API), it is because you have had one returned to you by some other API. In this case, the most helpful thing for you to know is how to get values from the observable, and the semantics of [subscription](GL)
+
+> The first thing to know is, an observable doesn't "do" anything.
+
+An observable, in fact, is _not_ a "stream". An observable is a template for [subscription](GL), through which your code (the [consumer](GL)), will tell a [producer](GL) to start pushing values to it by subscribing to an observable, creating a [subscription](GL).
+
+## Basic Subscription
+
+```ts
+const source$ = getSomeObservable();
+
+source$.subscribe(value => {
+    // do something with each value as it is pushed to your code
+    console.log(value);
+});
+```
+
+The most basic form of subscription is to subscribe to the observable with a single function as the argument to [`subscribe`](API). This will execute the observable's initialization function and create the [subscription](GL).
+
+## Subscription With An Observer
+
+```ts
+const source$ = getSomeObservable();
+
+source$.subscribe({
```

Comment:
Would it be worth also showing an example using the arrow function syntax?

And then point out the key difference between the named function version and the arrow function version ie how *this* is treated, especially if the next operation updates local state.